### ISSUE TITLE
[Bug][A11y][Chip] Group chips together

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1062]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14

### OS Version

iOS 26

### App version

0.6.0

### App build number

.

### App build type

beta

### Describe the issue

Chips are a kind of components that should be grouped when relating to the same setting or feature.
The group should be labelled in a way that we understand the concept of filter or suggestion (for example "filter by type of device").

### Expected Behavior

`.accessibilityElement(chldren: .contain)` is used to group controls.

Use the visible label (if present) to construct the `accessibilityLabel`

### Accessibility settings

Voice Over
